### DISCUSSION TITLE
Save behaviour in `dirsheet`

### DISCUSSION THREAD

**Discussion by proItheus** [saulpw/visidata#2163] [Q&A] [answered]
Any change in `dirsheet` is instantly committed to filesystem, without my executing `commit change` command.

[![asciicast](https://asciinema.org/a/iEAmZYerrGDBdgYnkB3u8jDoI.svg)](https://asciinema.org/a/iEAmZYerrGDBdgYnkB3u8jDoI)

I'm not sure if it's a bug, or if there are some relevant options I missed?

The version is `v2.11.1`

**Comment by midichef** [comment, marked as answer]
Just a note to point out that @anjakefala has answered in detail elsewhere, that this bug has been fixed in `develop`: https://github.com/saulpw/visidata/issues/2165#issuecomment-1847766965

**Reply by anjakefala** [reply]
I am wondering if my DirSheet guide PR is why there is a recent opening of these issues!
